File: app/src/main/java/com/xfq/easytest/Result.java
package com.xfq.easytest;

import android.os.Parcel;
import android.os.Parcelable;

import org.litepal.crud.LitePalSupport;

import java.util.List;

class Result extends LitePalSupport implements Parcelable {
    private Integer id;
    private String question;
    public static final Creator<Result> CREATOR = new Creator<Result>() {
        @Override
        public Result createFromParcel(Parcel in) {
            return new Result(in);
        }

        @Override
        public Result[] newArray(int size) {
            return new Result[size];
        }
    };
    private List<String> correctnessList;

    Result() {
    }

    protected Result(Parcel in) {
        if (in.readByte() == 0) {
            id = null;
        } else {
            id = in.readInt();
        }
        question = in.readString();
    }

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getQuestion() {
        return question;
    }

    public void setQuestion(String question) {
        this.question = question;
    }

    public List<String> getCorrectnessList() {
        return correctnessList;
    }

    public void setCorrectnessList(List<String> correctnessList) {
        this.correctnessList = correctnessList;
    }

    @Override
    public int describeContents() {
        return 0;
    }

    @Override
    public void writeToParcel(Parcel dest, int flags) {
        if (id == null) {
            dest.writeByte((byte) 0);
        } else {
            dest.writeByte((byte) 1);
            dest.writeInt(id);
        }
        dest.writeString(question);
    }
}

File: app/src/main/java/com/xfq/mwords/Result.kt
package com.xfq.mwords

import org.litepal.crud.LitePalSupport
import java.util.*


class Result : LitePalSupport() {
    var id: Int? = null
        get() = field
        set(value) {
            field = value
        }

    var user: String? = null
        get() = field
        set(value) {
            field = value
        }

    var unit: String? = null
        get() = field
        set(value) {
            field = value
        }

    var time: Int? = null
        get() = field
        set(value) {
            field = value
        }

    var help: Int? = null
        get() = field
        set(value) {
            field = value
        }

    var customHelp: Int? = null
        get() = field
        set(value) {
            field = value
        }

    var createdAt: Date? = null
        get() = field
        set(value) {
            field = value
        }
}